tests/flow/test_bulk_insertion.py

```diff
@@ -0,0 +1,127 @@
+import os
+import sys
+import unittest
+from .disposableredis import DisposableRedis
+from redisgraph import Graph, Node, Edge
+from base import FlowTestsBase
+sys.path.append(os.path.dirname(os.path.abspath(__file__)) + '/../../demo/bulk_insert/')
+
+import click
+from click.testing import CliRunner
+
+from bulk_insert import bulk_insert
+
+redis_con = None
+port = None
+graphname = None
+
+def redis():
+    return DisposableRedis(loadmodule=os.path.dirname(os.path.abspath(__file__)) + '/../../src/redisgraph.so')
+
+class GraphBulkInsertFlowTest(FlowTestsBase):
```

Comment:
Does this test actually uses batching, or does the entire graph fits into a single batch,
maybe it's worth exposing the batch size such that we test graph bulk insert when batching is being used.